### DISCUSSION TITLE
React Native support

### DISCUSSION THREAD

**Discussion by JessedeJonge** [pocketbase/pocketbase#464] [Q&A] [answered]
Hi all,

Great library! Am currently using Supabase but would love to use this for my new project. Any chance the Javascript SDK works with React Native / Expo ? For Supabase [you can initialize the client so it will store it tokens in AsyncStorage](https://supabase.com/docs/guides/with-expo#initialize-a-react-native-app). Is there any need to do this for Pocketbase too? Or an alternative?

**Comment by ganigeorgiev** [comment, marked as answer]
I haven't tested it, but there is no reason why the JavaScript SDK shouldn't work with React Native.

If you need custom AuthStore you can always extend the existing one (see https://github.com/pocketbase/js-sdk#authstore).

**Reply by JessedeJonge** [reply]
Nice, this is even better, so I can write the logic per function to even use [react-native-mkkv for example](https://github.com/mrousavy/react-native-mmkv). Might be a good idea to add some examples for this in the docs (with asyncstorage and/or mkvv)?
